plutus-book/doc/endmatter.adoc

```diff
@@ -8,48 +8,165 @@
+<<validator-script,validator script>> to check the validity of a
+<<script-input,script input>>.
 
-Extended UTxO ::
-The ledger model on which the Plutus platform is based. Extends the traditional UTxO model
-with data scripts and transaction information
+[[extended-utxo]]Extended UTXO ::
+The ledger model on which the <<plutus-platform,Plutus platform>> is based. 
+Extends the traditional <<utxo,UTXO>> model
+with <<data-script,data scripts>> and <<tx,transaction>> information
 
-Fungible Token ::
-A token that is interchangeable with other tokens.
+[[fungible-token]]Fungible Token ::
+A <<custom-token,token>> that is interchangeable with other tokens.
 
-Mockchain ::
-A Cardano blockchain emulator which faithfully implements the Extended UTxO model.
+[[mockchain]]Mockchain ::
+A Cardano blockchain emulator which faithfully implements the 
+<<extended-utxo,Extended UTXO model>>.
 
 Mock wallet ::
-A wallet emulator which is used to interact with the mockchain
+A wallet emulator which is used to interact with the <<mockchain,mockchain>>.
 
-Non-fungible Token ::
-A token that is not interchangeable with other tokens.
+[[non-fungible-token]]Non-fungible Token ::
+A <<custom-token,token>> that is not interchangeable with other tokens.
 
-Off-chain code ::
+[[off-chain1]]Off-chain code ::
 Code written as part of a smart contract which executes off the chain, usually in a user's wallet.
 
-On-chain code ::
-Code written as part of a smart contract which executes on the chain during transaction validation.
+[[on-chain]]On-chain code ::
+Code written as part of a smart contract which executes on the chain during
+<<tx,transaction>> <<validator-script,validation>>.
 
+[#plutus-core]
 Plutus Core ::
-A small functional programming language designed to run as on-chain code.
+A small functional programming language designed to run as <<on-chain,on-chain>> code.
 
 Plutus IR ::
-An intermediate language that compiles to Plutus Core, for use as a target language for compiler writers.
+An intermediate language that compiles to <<plutus-core,Plutus Core>>, 
+for use as a target language for compiler writers.
 
-Plutus Platform ::
+[[plutus-platform]]Plutus Platform ::
 The combined software support for writing smart contracts, including:
-- libraries for writing off-chain code in Haskell.
-- libraries for writing on-chain code in Plutus Tx.
-- emulator support for testing smart contracts.
+- Libraries for writing <<off-chain1,off-chain>> code in Haskell.
+- Libraries for writing <<on-chain,on-chain>> code in <<plutus-tx,Plutus Tx>>.
+- Emulator support for testing smart contracts.
 
 Plutus Playground ::
 A browser based environment for writing and testing Plutus contracts.
 
-Plutus Tx ::
-A subset of Haskell which is compiled into Plutus Core.
+[[plutus-tx]]Plutus Tx ::
+A subset of Haskell which is compiled into <<plutus-core,Plutus Core>>.
+
+[[pubkey-address]]Public Key Address ::
+Possible address type for a <<transaction-output,transaction output>>.
```

Comment:
You don't need to do this: if you define the anchor with a reftext you can just refer to it once. I'll make a PR fixing this.

Reply:
I did that on purpose, because I am using different ref texts for the same anchor sometimes and thought it would be more explicit and less confusing this way. But by all means, go ahead. Just keep the changed reftexts in place.

Reply:
PR here, feel free to accept it or not. I think otherwise there's a *lot* of repeating yourself in the common case.